Jenkinsfile

```diff
@@ -34,9 +34,19 @@ if (needSplittingFromWorkspace) {
 
 branches['CI'] = {
   stage('CI') {
-    node('maven-11') {
-      checkout scm 
-      sh "mvn verify --no-transfer-progress -DskipTests -P jenkins-release"
+    retry(count: 2, conditions: [kubernetesAgent(handleNonKubernetes: true), nonresumable()]) {
```

Comment:
Consistent with core and BOM: even though in practice these are Kubernetes agents, we respect the contract provided by the infrastructure team by not relying on this implementation detail.